stock_picking_operations_multilang/tests/test_stock_picking_operations_multilang.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 Quartilte Limited (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestPickingOperationsLanguage(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        ja = (
+            cls.env["res.lang"]
+            .with_context(active_test=False)
+            .search([("code", "=", "ja_JP")])
+        )
+        cls.env["base.language.install"].create({"lang_ids": ja.ids}).lang_install()
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "Test Partner", "lang": "ja_JP"}
+        )
+        cls.picking_type_in = cls.env.ref("stock.picking_type_in")
+        cls.picking = cls.env["stock.picking"].create(
+            {
+                "partner_id": cls.partner.id,
+                "picking_type_id": cls.picking_type_in.id,
+            }
+        )
```

Comment:
Just to add shortcuts.

```suggestion
        )
        cls.warehouse = cls.picking_type_in.warehouse_id
        cls.report =  cls.env.ref("stock.action_report_picking")
```